ttnn/cpp/ttnn/distributed/api.cpp

```diff
@@ -140,7 +144,7 @@ std::vector<int> get_t3k_physical_device_ids_ring() {
     return physical_device_ids;
 }
 
-std::vector<Device*> distribute_tensor_to_mesh(const Tensor& tensor, MeshDevice& mesh_device) {
+std::vector<Device*> get_devices_for_tensor(const Tensor& tensor, MeshDevice& mesh_device) {
```

Comment:
minor but not a huge fan of the name change. maybe something like:
`map_tensor_to_devices` or `get_mapped_devices`?